spotirec.py

```diff
@@ -253,7 +256,7 @@ def parse_seed_info(seeds):
     :param seeds: seed data as a string or a list
     """
     if len(shlex.split(seeds) if type(seeds) is str else seeds) > 5:
-        print('Please enter at most 5 seeds')
+        logging.info('Please enter at most 5 seeds')
```

Comment:
Should this not be an error? The request accepts at most 5 seeds, and Spotirec will exit if more are provided.

Reply:
Fixed in d3df488